tb_plugin/torch_tb_profiler/profiler/node.py

```diff
@@ -72,6 +76,9 @@ def __init__(self, name, start_time, end_time, type, tid, external_id=None, devi
         self.self_device_duration = self_device_duration
         self.memory_records = []
         # self.parent_node = None
+        self.tc_eligible = False
+        self.tc_self_duration = 0
```

Comment:
add comments about the meaning of each field

Reply:
Thanks. Done.